Userland/Shell/Builtin.cpp

```diff
@@ -1198,9 +1198,12 @@ ErrorOr<int> Shell::builtin_wait(Main::Arguments arguments)
 ErrorOr<int> Shell::builtin_unset(Main::Arguments arguments)
 {
     Vector<DeprecatedString> vars;
+    bool posix_only_variables = false;
```

Comment:
nit: Not clear if this is `posix (only variables)` or `(posix only) variables` at first read, and both kind of make sene. Maybe `posix_unset_only_variables`? Or maybe `bool unset_only_variables = false; // POSIX-only`?